docs/cluster-management/configure-api.md

```diff
@@ -253,6 +253,37 @@ scm:
         oauthClientSecret: YOUR-APP-SECRET
 ```
 
+### Notifications Plugin
+
+Currently, only an email notifications plugin is supported. Additional notifications plugins may be added in the future.
+
+Additional notifications plugins can be configured using the following format:
+
+```yaml
+# config/local.yaml
+notifications:
+    email:
+        host: smtp.yourhost.com
+        port: 25
+        from: example@email.com
+    webhook: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXX
+```
+
+#### Email Notifications
+
+Configure the SMTP server and sender address that email notifications will be sent from.
+
+```yaml
+# config/local.yaml
+notifications:
+    email:
+        host: smtp.yourhost.com
+        port: 25
+        from: example@email.com
+```
+
+Configurable authentication settings have not yet been built, but can easily be added. We’re using the nodemailer package to power emails, so authentication features will be similar to any typical nodemailer setup.
```

Comment:
We can say if they want to contribute then go to https://github.com/screwdriver-cd/notifications-email